packages/u/urlscan/package.yml

```diff
@@ -1,15 +1,20 @@
 name       : urlscan
-version    : 0.9.4
-release    : 7
+version    : 1.0.1
+release    : 8
 source     :
-    - https://github.com/firecat53/urlscan/archive/0.9.4.tar.gz : fc76c3c8a852f1fbdde69faa336184b7f060b33e8e4573d6ffa8790ed6f429b6
+    - https://files.pythonhosted.org/packages/b7/7d/984994a32b261cc3a72d3bfb0b8c0de4f786682128dc659a5c5e02dfc48c/urlscan-1.0.1.tar.gz: e0ec986e5aa2da57dd2face8692116d80af173d4eb56a78e4fd881731113307f
```

Comment:
Can you change the source to `https://files.pythonhosted.org/packages/source/u/urlscan/urlscan-1.0.1.tar.gz` <- more readable and doesn't break `go-task bump` formatting.

Reply:
ok